packages/graphback-core/package.json

```diff
@@ -44,12 +44,13 @@
     "graphql-compose": "7.18.1",
     "graphql-fields-list": "2.1.3",
     "graphql-metadata": "0.7.0",
+    "graphql-scalars": "1.2.2",
     "graphql-subscriptions": "1.1.0",
-    "graphql-type-json": "0.3.2",
     "pino": "6.3.2",
     "pluralize": "8.0.0"
   },
   "peerDependencies": {
-    "graphql": "^14.0.0 || ^15.0.0"
+    "graphql": "^14.0.0 || ^15.0.0",
+    "mongodb": "3.5.9"
```

Comment:
This means user must use mongodb version 3.5.9? Maybe it should be less restrictive and allow a minimum version.

Reply:
This is what we have for the mongo runtime too:

https://github.com/aerogear/graphback/blob/ff8ea5b04b1414625d4d7daca203ecad58e06777/packages/graphback-runtime-mongodb/package.json#L49